scripts/sample-reads-randomly.py

```diff
@@ -86,7 +86,7 @@ def get_parser():
 
     parser.add_argument('filenames', nargs='+')
     parser.add_argument('-N', '--num_reads', type=int, dest='num_reads',
-                        default=DEFAULT_NUM_READS)
+                        default=DEFAULT_NUM_READS, help='samples the number of sequences or pairs specified with -N')
```

Comment:
khmer enforces the PEP8 style guide for all Python code. This includes a limit of 79 characters per line. You can wrap strings across multiple lines as shown below.

```python
function(arg1, arg2, arg3='if the string is really long and has a lot of words'
         ' you can wrap it across multiple lines like this')
```

You can also check for any code style violations with `make pep8` before committing.